pkg/query-service/model/v3/v3.go

```diff
@@ -229,13 +229,14 @@ type AggregateAttributeRequest struct {
 type TagType string
 
 const (
-	TagTypeTag      TagType = "tag"
-	TagTypeResource TagType = "resource"
+	TagTypeTag                  TagType = "tag"
+	TagTypeResource             TagType = "resource"
+	TagTypeInstrumentationScope TagType = "instrumentation_scope"
```

Comment:
nit: this is not used anywhere.

Reply:
Ahh my bad it should have been `scope`, have updated it

it will allow these queries
```
{
    "id": "3d0fc1fc-08ba-4cb9-ab98-c38cb1dfb460",
    "key": {
        "key": "scope_attr",
        "dataType": "string",
        "type": "scope",
        "isColumn": false,
        "isJSON": false
    },
    "op": "in",
    "value": "attr_val"
}
```